spec/features/admin_claim_check_spec.rb

```diff
@@ -80,6 +80,15 @@
       expect(page).to have_content(user.full_name)
     end
 
+    context "When the claim has had PII removed" do
+      let!(:claim_with_pii_removed) { create(:claim, :rejected, :pii_removed) }
+
+      scenario "User can see when a claim has had PII removed" do
```

Comment:
Might be worth checking that the payroll gender stuff doesn't get mentioned in this scenario too

Reply:
It might actually make more sense to remove the payroll gender warning entirely if a claim is rejected - at that point an operator cannot take any further action, so why warn them?